rs/tests/driver/src/driver/bootstrap.rs

```diff
@@ -286,25 +284,24 @@ pub fn setup_and_start_vms(
             let conf_img_path = PathBuf::from(&node.node_path).join(CONF_IMG_FNAME);
             match InfraProvider::read_attribute(&t_env) {
                 InfraProvider::K8s => {
-                    let url = format!(
-                        "{}/{}",
-                        tnet_node.config_url.clone().expect("missing config_url"),
-                        CONF_IMG_FNAME
+                    let command = format!(
```

Comment:
It would be good to have a few sentences of documentation here about what's going on.